sdk/lib/3rdparty/freetype/src/psaux/psintrp.c

```diff
@@ -526,14 +526,14 @@
 
     CF2_HintMaskRec   hintMask;
 #ifdef __REACTOS__
-    CF2_GlyphPathRec *glyphPath = malloc(sizeof(CF2_GlyphPathRec));
-    if (!glyphPath) return;
-/* Ugly but it allows us to reduce the diff */
-#define glyphPath (*glyphPath)
+    CF2_GlyphPathRec *glyphPath = malloc(sizeof(*glyphPath));
```

Comment:
Please restore the NULL-check + early return.

Reply:
Restored.